floris/tools/optimization/scipy/yaw.py

```diff
@@ -385,10 +418,6 @@ def minimum_yaw_angle(self):
 
     @minimum_yaw_angle.setter
     def minimum_yaw_angle(self, value):
-        if not hasattr(self, "maximum_yaw_angle"):
-            self._set_opt_bounds(value, 25.0)
-        else:
-            self._set_opt_bounds(value, self.maximum_yaw_angle)
```

Comment:
Removed the calls to `_set_opt_bounds` because this would overwrite bounds that were previously set by the optional `bnds` argument. @bayc please check if this is ok! There might be another way to handle this.

Reply:
I think this works.